src/plugins/usage_collection/tsconfig.json

```diff
@@ -24,6 +24,9 @@
     "@kbn/logging",
     "@kbn/ebt",
     "@kbn/core-saved-objects-server",
+    "@kbn/core-saved-objects-api-server-internal",
```

Comment:
Why do we need this internal Core module import (`@kbn/core-saved-objects-api-server-internal`)? Is that only to stub `getCurrentTime` during the integration tests?

Reply:
Yes I'm afraid so, it bothers me too.
Perhaps I can try to find a cleaner way to insert old counters for testing purposes.

Reply:
In https://github.com/elastic/kibana/pull/187665/commits/f90542c2ac8055fd1164da3a4dd1d15b31401f4f I used the standard `incrementCounter` to create the counters, and then I used `esClient.updateByQuery()` to modify their `updated_at` dates. Not ideal, but cleaner than the mock IMO, and addresses your feedback above.